README.md

```diff
@@ -27,7 +27,19 @@ The `Import Address Table` (IAT) patching technique is a documented approach tha
 
 To use the mock behavior for Win32 API (or other DLL) module functions, you'll first need to build the library. Once that's done, you can include the library `gmock-win32.h` header immediately after the `gmock/gmock.h` header in your test code. Several macros are then available to help you achieve the desired mock behavior:
 
-#### 1. Create function mocks by using `MOCK_STDCALL_FUNC` / `MOCK_CDECL_FUNC` macro (by defining them in global scope)
+#### 1. First, we need to initialize the library by using `initialize` / `uninitialize` or `init_scope` initialization wrapper. Typically, it can be done in `main` function before initializing `GTest`:
+
+```cpp
+int main(int argc, char* argv[])
+{
+    const gmock_win32::init_scope gmockWin32{ };
+
+    testing::InitGoogleTest(&argc, argv);
+    return RUN_ALL_TESTS();
+}
+```
+
```

Comment:
If there is `testing::InitGoogleTest`, why don't we do the same. This is expected.